src/main/kotlin/no/nav/syfo/varsel/AktivitetskravVarselPlanner.kt

```diff
@@ -27,70 +26,95 @@ class AktivitetskravVarselPlanner(
 
     private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.varsel.AktivitetskravVarselPlanner")
     private val varselUtil: VarselUtil = VarselUtil(databaseAccess)
-    private val sykeforlopService: SykeforlopService = SykeforlopService()
 
     override suspend fun processOppfolgingstilfelle(aktorId: String, fnr: String) = coroutineScope {
         val oppfolgingstilfellePerson = syfosyketilfelleConsumer.getOppfolgingstilfelle(aktorId)
+        log.info("-$name-: oppfolgingstilfellePerson for -$aktorId- aktor id er -$oppfolgingstilfellePerson-")
```

Comment:
Jeg tror vi egentlig ikke skal logge aktorId i prod, siden det kan knyttes til personopplysninger, og denne kontektsen, sykefraværsdata. Det finnes en metode for å sende til logger som kun teamet har tilgang til,
men før vi har satt opp det bør vi kanskje unngå å sende aktorId til åpne logger.

Reply:
Burde skrive det i kommentar tidligere, men tenkte å logge den bare i en kort periode(1 uke) for å kunne se på konkrete oppfølgingstilfeller sammen med Lisa. Så skulle jeg fjerne aktorId fra logging.